commit f7468a7160e1086a7960d6fea8735f49c3c3c900
Author: Tony Lindgren <tony@atomide.com>
Date:   Wed Aug 8 01:07:06 2018 -0700

    bus: ti-sysc: Fix no_console_suspend handling
    
    [ Upstream commit 4f3530f4a41d49c41015020cd9a5ed5c95b5d2db ]
    
    If no_console_suspend is set, we should keep console enabled during suspend.
    Lets fix this by only producing a warning if we can't idle hardware during
    suspend.
    
    Fixes: ef55f8215a78 ("bus: ti-sysc: Improve suspend and resume handling")
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/bus/ti-sysc.c b/drivers/bus/ti-sysc.c
index b31bf03ea497..4576a1268e0e 100644
--- a/drivers/bus/ti-sysc.c
+++ b/drivers/bus/ti-sysc.c
@@ -1175,10 +1175,10 @@ static int sysc_child_suspend_noirq(struct device *dev)
 	if (!pm_runtime_status_suspended(dev)) {
 		error = pm_generic_runtime_suspend(dev);
 		if (error) {
-			dev_err(dev, "%s error at %i: %i\n",
-				__func__, __LINE__, error);
+			dev_warn(dev, "%s busy at %i: %i\n",
+				 __func__, __LINE__, error);
 
-			return error;
+			return 0;
 		}
 
 		error = sysc_runtime_suspend(ddata->dev);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1268_f7468a7160e1086a7960d6fea8735f49c3c3c900.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1268_f7468a7160e1086a7960d6fea8735f49c3c3c900.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
